### ISSUE TITLE
Gridcoin 3.5.9.0 Error on Win 7 x64 Pro  ** Critical Error **

### ISSUE THREAD

**Issue by TG13088** [gridcoin-community/Gridcoin-Research#359]
I just encountered a failure during an automated upgrade to the Gridcoin Wallet App on Win 7 x64 Pro.  The upgrade failed trying to update \Program Files\GridcoinResearch\gridcoinresearch.exe with insufficient permissions.  Basically it is calling for SYSTEM permissions.  I cannot replace the file, delete it nor delete the directory it is in.  I tried using the latest MSI on the Gridcoin US site and it cannot touch the file either.  This is a *critical* error not because of the wallet failure but because of the error it introduces via the automated distibution.

**Comment by iFoggz** [comment]
your suppose to do upgrade while elevated administrator privileges are you not?
You tried in app upgrade as well?
Worst comes to worst u may have to go the safe mode route to remove it but idk any other reason for a permissions problem

**Comment by TG13088** [comment]
The Wallet app runs with Administrative privileges as configured.  The Uninstall/ReInstall process using the MSI file fails to delete the file.  Right now, the Wallet is down due to the inability to "touch" the file or directory using *ANY* built-in Windows facility.  I have to find a utility to better understand the permissions used on the directory and the file itself but there are no known issues that I see.  I cannot even change the ownership of the file or directory and when queried, it indicates that it is *unable to identify owner*.

**Comment by iFoggz** [comment]
https://support.microsoft.com/en-us/help/2623670/-access-denied-or-other-errors-when-you-access-or-work-with-files-and-folders-in-windows may have some methods u can try.

also seems like the file may be locked. are u sure its not running in a process or have a thread running? could try a restart if the link above doesn't help.

**Comment by TG13088** [comment]
Closing issue.  Appears that the file may have been "locked" by the ABEND'ed Wallet task.  A reboot seems to have cleared the problem and successfully restarted it.  It came up as 3.5.8.9 so I used the Upgrade Client tab to perform the upgrade.  That successfully processed the 3.5.9.0 upgrade so everything is up to date.  Also, it is official ... **I HATE COMPUTERS** :-P  Many Thanks for everyone's help!
